Winleafs-Server/Winleafs.Server.Api/Controllers/SpotifyController.cs

```diff
@@ -0,0 +1,148 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Serilog;
+using System;
+using System.Threading.Tasks;
+using Winleafs.Server.Services.Exceptions;
+using Winleafs.Server.Services.Helpers;
+using Winleafs.Server.Services.Interfaces;
+
+namespace Winleafs.Server.Api.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class SpotifyController : BaseApiController
+    {
+        private IUserService _userService;
+        private ISpotifyService _spotifyService;
+
+        public SpotifyController(DbContext context, IUserService userService, ISpotifyService spotifyService) : base(context)
+        {
+            _userService = userService;
+            _spotifyService = spotifyService;
+        }
+
+        /// <remarks>
+        /// Must be GET due to Spotify authorization standards
+        /// </remarks>
+        [HttpGet]
+        [Route("authorize/{applicationId}")]
+        public async Task<IActionResult> Authorize(string applicationId)
+        {
+            Guid guidOutput;
+            if (string.IsNullOrWhiteSpace(applicationId) || !Guid.TryParse(applicationId, out guidOutput))
```

Comment:
out params can now be inlined. Something like `out var guidOutput`. Unsure if it works in this situation